system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -554,13 +545,7 @@ construct_runtime!(
 		BridgePolkadotMessages: pallet_bridge_messages::<Instance1> = 53,
 		XcmOverBridgeHubPolkadot: pallet_xcm_bridge_hub::<Instance1> = 54,
 
-		// Ethereum bridge pallets.
-		EthereumInboundQueue: snowbridge_pallet_inbound_queue = 80,
-		EthereumOutboundQueue: snowbridge_pallet_outbound_queue = 81,
-		EthereumBeaconClient: snowbridge_pallet_ethereum_client = 82,
-		EthereumSystem: snowbridge_pallet_system = 83,
```

Comment:
These were never used by anyone so I don't think we need to save their indices, we can safely reuse index 80-83